### ISSUE TITLE
Missing parser.sh file in the repo?

### ISSUE THREAD

**Issue by Ziver** [ktraister/sonarqube-bash-plugin#4]
Hi,
I've been trying to run the maven tests but it fails very quickly in the TokenizerSensor class where it is trying to look for a parser.sh file that does not seem to exist in the repo.

**Comment by ktraister** [comment]
Yeah, this started out as a powershell plugin that I started to port over to bash using the tool ShellShock as a backend. I don't remember how far I got in that effort, as it was for a previous employer
